src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -44,14 +44,20 @@ export interface IChoiceGroupOption {
   imageSize?: { width: number, height: number };
 
   /**
-   * Whether the option is disabled or not.
+   * Whether the options is checked or not.
    * @defaultvalue false
    */
-  isDisabled?: boolean;
+  isChecked?: boolean;
```

Comment:
just calling this out but the same bug is here; this probably should be "checked" and not isChecked. Don't have to update it in this PR but it would be nice in some future update :)